### PR TITLE
Virtualenv fixes

### PR DESCRIPTION
**Pull request by akshayjshah**
Fix some small bugs with the way we manage and ignore Python virtualenvs.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -9,6 +9,7 @@
 /cover
 /cover.html
 /cover.out
+/env
```
Comment (peter-edge) [inline .gitignore]: Is this not covered by `/internal/examples/env`?

File: .dockerignore [hunk 2]
```diff
@@ -9,6 +9,7 @@
 cover
 cover.html
 cover.out
+env
```
Comment (peter-edge) [inline .dockerignore]: Should be covered by `internal/examples/env`?

**Comment by akshayjshah** [comment]
@peter-edge over-eager ignores removed

**Comment by peter-edge** [comment]
LGTM

**Pull request merged**